src/Sulu/Bundle/AdminBundle/Metadata/MetadataProviderInterface.php

```diff
@@ -13,5 +13,7 @@
 
 interface MetadataProviderInterface
 {
+    public const SERVICE_TAG = 'sulu_admin.metadata_provider';
+
```

Comment:
not a fan of this constants would remove it, its are literals which only matter during service registrations and has nothing todo with the application or domain interfaces.

Reply:
Maybe moving it to the bundle instead? Because I still think they could be helpful. Especially when we're moving to a php based configuration for services.